org.bridgedb.rdf/resources/IdentifiersOrgDataSource.ttl

```diff
@@ -1,4 +1,4 @@
-@prefix bridgeDB: <http://vocabularies.bridgedb.org/ops#> .
+.prefix bridgeDB: <http://vocabularies.bridgedb.org/ops#> .
```

Comment:
That's a typo, not?

Reply:
Looks like a type - still built and worked with it though! Feel free to change